README.md

```diff
@@ -494,6 +494,25 @@ If you don't have a bridging header, you can go to your target's `Build Settings
 
 ** Comparison with RAC with respect to v3.0-RC.1
 
+### Is RxSwift Functional Reactive Programming?
+
+Why this question is important? Because RxSwift is **NOT** *Functional Reactive Programming*. This clarification is important for 2 big reasons:
+
+* the Rx world is not time-continuous and doesn't have a denotative semantics, so all fundamentals of FRP are missing
```

Comment:
"so all fundamentals of FRP are missing" This sounds negative for users. If I were to read this out of context, and people will, it sounds we are lacking something vital instead of it shouldn't being there in the first place :)